code/modules/crew_objectives/security_objectives.dm

```diff
@@ -0,0 +1,33 @@
+/*				SECURITY OBJECTIVES				*/
+
+/datum/objective/crew/enjoyyourstay
+	explanation_text = "Welcome aboard. Enjoy your stay."
+	jobs = "headofsecurity,securityofficer,warden,detective"
+	var/list/edglines = list("Welcome aboard. Enjoy your stay.", "You signed up for this.", "Abandon hope.", "The tide's gonna stop eventually.", "Hey, someone's gotta do it.", "No, you can't resign.", "Security is a mission, not an intermission.")
+
+/datum/objective/crew/enjoyyourstay/New()
+	. = ..()
+	update_explanation_text()
+
+/datum/objective/crew/enjoyyourstay/update_explanation_text()
+	. = ..()
+	explanation_text = pick(edglines)
+
+/datum/objective/crew/enjoyyourstay/check_completion()
+	explanation_text = "Enforce Space Law to the best of your ability."
+	if(owner && owner.current)
+		if(owner.current.stat != DEAD)
+			return TRUE
+	return FALSE
+
+/datum/objective/crew/justicecrew
+	explanation_text = "Ensure there are no innocent crew members in the brig when the shift ends."
```

Comment:
This will need to be changed.

Reply:
Actually, does it count the shuttle tiles that give redtext as "in the brig" as well? Could still work.

And if you really wanted to go ham: give security greentext that requires all antags redtext.

Reply:
Changed it.